pkgs/top-level/all-packages.nix

```diff
@@ -30157,8 +30155,7 @@ with pkgs;
 
   emacsMacport = callPackage ../applications/editors/emacs/macport.nix {
     withMacport = true;
-
-    gconf = null;
+    gconf = gnome2.GConf;
```

Comment:
Do we really need gconf on Darwin?

Reply:
This is the way.
gconf is not on the toplevel. Then it needs to receive an explicit value, whether if the underlying code will use it or not.

Reply:
No, I'm asking whether gconf is in any way required for macport emacs (which only runs on Darwin).

Reply:
I can't know for sure. The Nix code by itself filters gconf when building for Darwin, so my guess is that it isn't needed.

Reply:
That's what I mean. Just drop the override then.

Reply:
I am thinking about removing gconf entirely, given that it became obsolete/deprecated.